core/api/api_views.py

```
@@ -720,34 +731,29 @@ def generate_report(request):
+        # Map serializer choices to enums
+        # Expect serializer to emit canonical strings like 'CLAIMS_SUMMARY', 'CSV', and date objects
+        report_type_value = v.get('report_type')
+        format_value = v.get('format_type', 'CSV')
+        start_date = v.get('start_date')
+        end_date = v.get('end_date')
 
-        # Validate and convert enums
         try:
-            report_type_enum = ReportType(report_type_raw)
+            report_type_enum = ReportType(report_type_value)
         except Exception:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)